operator/redisfailover/service/generator.go

```diff
@@ -680,6 +694,11 @@ func createSentinelExporterContainer(rf *redisfailoverv1.RedisFailover) corev1.C
 		},
 		Resources: resources,
 	}
+
+	if rf.Spec.Sentinel.Exporter.Env != nil {
```

Comment:
Why is this needed? I think its already done before, in the line 673

Reply:
Oh yes. removed!